prime-router/src/main/kotlin/fhirengine/translation/hl7/FhirToHl7Converter.kt

```diff
@@ -164,7 +164,7 @@ class FhirToHl7Converter(
                     }
 
                     // A value
-                    element.value.isNotEmpty() && element.hl7Spec.isNotEmpty() -> {
+                    !element.value.isNullOrEmpty() && element.hl7Spec.isNotEmpty() -> {
```

Comment:
Maybe this should go in a follow up, but I think the water is still a little muddy here on `null` vs an empty map.  The new state going forward is that every where now needs to check for both?

Fine by be if you drop this in the tech debt ticket.

I would think that we would want a new test though that asserts this branch when its null and when its an empty map

Reply:
Worth pointing out this is `value` and not `valueSet`, but agree there's some room to explore the handling of `value` further. We could either explore the open questions from this ticket in a new ticket, or perhaps add it to #10531.

Reply:
Right, but we changed the default value for it?  Fine deferring thinking through this more, but I think the change warrants a new test